Gustav/setrun.py

```diff
@@ -0,0 +1,524 @@
+    #rundata.gaugedata.gauges.append([8, -88.0583, 30.66,
+    #                                 rundata.clawdata.t0,
+    #                                 rundata.clawdata.tfinal])                              
+                                     
+    #Mobile State Docks, AL (8737048)
+    #rundata.gaugedata.gauges.append([9, -88.04, 30.705,
+    #                                 rundata.clawdata.t0,
+    #                                 rundata.clawdata.tfinal])
+    
+    #Shell Beach (8761305)
+    rundata.gaugedata.gauges.append([6, -89.6733, 29.9683,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])                                 
+                                    
+    # Force the gauges to also record the wind and pressure fields
+    rundata.gaugedata.aux_out_fields = [4, 5, 6]
+
+    # ------------------------------------------------------------------
+    # GeoClaw specific parameters:
+    # ------------------------------------------------------------------
+    rundata = setgeo(rundata)
+
+    return rundata
+    # end of function setrun
+    # ----------------------
+
+
+# -------------------
+def setgeo(rundata):
+    """
+    Set GeoClaw specific runtime parameters.
+    For documentation see ....
+    """
+
+    geo_data = rundata.geo_data
+
+    # == Physics ==
+    geo_data.gravity = 9.81
+    geo_data.coordinate_system = 2
+    geo_data.earth_radius = 6367.5e3
+    geo_data.rho = 1025.0
+    geo_data.rho_air = 1.15
+    geo_data.ambient_pressure = 101.3e3
+
+    # == Forcing Options
+    geo_data.coriolis_forcing = True
+    geo_data.friction_forcing = True
+    geo_data.friction_depth = 1e10
+
+    # == Algorithm and Initial Conditions ==
+    # Note that in the original paper due to gulf summer swelling this was set
+    # to 0.28
+    geo_data.sea_level = 0.0
+    geo_data.dry_tolerance = 1.e-2
+
+    # Refinement Criteria
+    refine_data = rundata.refinement_data
+    refine_data.wave_tolerance = 1.0
+    refine_data.speed_tolerance = [1.0, 2.0, 3.0, 4.0]
+    refine_data.variable_dt_refinement_ratios = True
+
+    # == settopo.data values ==
+    topo_data = rundata.topo_data
+    topo_data.topofiles = []
+    # for topography, append lines of the form
+    #   [topotype, fname]
+    # See regions for control over these regions, need better bathy data for
+    # the smaller domains
+    
+    #topo_files = get_topo()
+    #for topo_file in topo_files:
+    #	topo_data.topofiles.append([4,1,5,rundata.clawdata.t0, rundata.clawdata.tfinal, topo_final])
+    #topo_data.topofiles.append([3,1,5,rundata.clawdata.t0, rundata.clawdata.tfinal,os.path.join(os.environ['CLAW'], 'geoclaw', 'scratch', 'NewOrleans_3s.tt3')])
+    clawutil.data.get_remote_file("http://www.columbia.edu/~ktm2132/bathy/gulf_caribbean.tt3.tar.bz2")
+    topo_path = os.path.join(scratch_dir, 'gulf_caribbean.tt3')
+    topo_path2 = os.path.join(scratch_dir, 'new_orleans.asc')
```

Comment:
You did not make mention as to where to get this topography in the `Readme.md`.  If you downloaded it from NOAA then you can just give the site and coordinate bounds.